supervision/detection/line_zone.py

```diff
@@ -197,6 +197,23 @@ def trigger(self, detections: Detections) -> Tuple[np.ndarray, np.ndarray]:
 
         return crossed_in, crossed_out
 
+    @staticmethod
+    def _cross_product(anchors: np.ndarray, vector: Vector) -> np.ndarray:
```

Comment:
We prefer methods like this to live in `utils.py`.

Please move it there and remove leading underscore `_`.

Reply:
done